pandas/core/generic.py

```diff
@@ -9162,6 +9162,12 @@ def first(self, offset) -> Self:
         3 days observed in the dataset, and therefore data for 2018-04-13 was
         not returned.
         """
+        # GH45908 & GH#52487
```

Comment:
I don't think we need the issue numbers here

Reply:
Thanks